01-prerequisites.md

```diff
@@ -10,12 +10,23 @@ This is the starting point for the instructions on deploying the [AKS Secure Bas
    >
    > * [Contributor role](https://docs.microsoft.com/azure/role-based-access-control/built-in-roles#contributor) is _required_ at the subscription level to have the ability to create resource groups and perform deployments.
    > * [User Access Administrator role](https://docs.microsoft.com/azure/role-based-access-control/built-in-roles#user-access-administrator) is _required_ at the subscription level since you'll be performing role assignments to managed identities across various resource groups.
+   >
+   > Run following CLI commands to understand current roles associated with Azure account.
+   >
+   >```azurecli-interactive
+   > az login
+   >```
+   >
+   >```azurecli-interactive
+   > az role assignment list --all --assignee <YOUR-LOGIN-ACCOUNT> --output json --query '[].{roleDefinitionName:roleDefinitionName, scope:scope}'
```

Comment:
I think this may have potential to cause confusion for the following reasons:

1. Contributor role _or higher_ is required. E.g. I have a classic administrator role on my subscription, which works fine for the purposes of this guide, but results in the above command returning empty.
2. We need both RBAC roles and Azure AD roles (User Access Administrator) in place. This command only checks the RBAC piece.